src/argilla/tasks/create_user.py

```diff
@@ -0,0 +1,57 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Optional
+
+import click
+
+from argilla.server.contexts import accounts
+from argilla.server.database import SessionLocal
+from argilla.server.models import User, UserRole
+from argilla.server.security.model import UserCreate
+
+
+def _show_created_user(user: User):
+    message = f"\nUsername: {user.username!r}" f"\nRole: {user.role.value!r}" f"\nFirst name: {user.first_name!r}"
+
+    if user.last_name:
+        message += f"\nLast name: {user.last_name!r}"
+
+    message += f"\nAPI-Key: {user.api_key!r}"
+
+    return message
+
+
+@click.command("create-user")
+@click.option("--first-name", prompt=True)
+@click.option("--username", prompt=True)
+@click.option(
+    "--role", prompt=True, type=click.Choice(UserRole.__members__, case_sensitive=False), default=UserRole.annotator
+)
+@click.option("--password", prompt=True, hide_input=True, confirmation_prompt=True)
+@click.option("--last-name")
```

Comment:
Possible changes/improvements:
- Remove the name of the command from `@click.command` because it takes the name of the function (but it's totally ok if you want to still use it).
- Follow the same order of the columns that we have in the database model (I have changed them to follow that order in the suggestion).
- Use `@click.password_option()` for the password option. It's exactly the same that you are using (but it's ok to be explicit if you want).
- Add `show_default=True` to ` --role` option (so it shows the default value if you pass `--help` to the task).

```suggestion
@click.command()
@click.option("--first-name", prompt=True)
@click.option("--last-name")
@click.option("--username", prompt=True)
@click.option(
    "--role",
    prompt=True,
    type=click.Choice(UserRole.__members__, case_sensitive=False),
    default=UserRole.annotator,
    show_default=True,
)
@click.password_option()
```

Reply:
> Possible changes/improvements:
> 
> * Remove the name of the command from `@click.command` because it takes the name of the function (but it's totally ok if you want to still use it).
Totally, I was doing some tests and I forgot remove the name.

> * Follow the same order of the columns that we have in the database model (I have changed them to follow that order in the suggestion).

I prefer keep the order only for prompted and required args. Since last_name is not requested in prompt and is an optional param, I would like keep at the end of function signature.

> * Use `@click.password_option()` for the password option. It's exactly the same that you are using (but it's ok to be explicit if you want).
+1 

> * Add `show_default=True` to ` --role` option (so it shows the default value if you pass `--help` to the task).
+1